File: exer10.c
/**
 * Exercício 10, lista de exercícios 06-03-2013
 * Programa que calcula a área de um círculo
 */

#include <stdio.h>
#include <math.h>

int main()
{

	float raio, area;

	printf("%s\n", "Programa que calcula a área de um círculo");
	printf("%s\n", "Escreva o raio do círculo:");
	scanf("%f", &raio);

	area = M_PI * pow(raio, 2);

	printf("%s: %f\n", "A área do círculo é de", area);

	return 0;
}

File: exer10.pas
Program AreaCirculo ;

Var
	raio, area: real;
 Begin
  	writeln('Programa que calcula a área de um círculo');
  	writeln('Escreva o raio do círculo:');
  	readln(raio);
  	
  	area := PI * sqr(raio);
  	
  	write('A área do círculo de raio ', raio, ' é de ', area);
 End.